### ISSUE TITLE
"Never transacted before" warning after successful ERC20 transfer

### ISSUE THREAD

**Issue by gbrew** [RabbyHub/Rabby#2273]
On arbitrum, using rabby, I sent some USDT (0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9) to an address which I saved in my whitelist. This worked fine, and the transaction shows up in the "transactions" view in rabby correctly.

I then tried to send USDC (Bridged, 0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8) to the same address. In the transaction confirmation screen, rabby gives a "Never transacted before" warning, but also shows the destination address as the whitelisted one I just sent to above.

What is the intended behavior of this warning? Does the fact that I'm now trying to transfer a different ERC20 token make a difference here? I can understand not counting all ERC20 transfers as scam tokens could really mess with things, but the behavior I'm seeing is disconcerting.

**Comment by vvvvvv1vvvvvv** [comment]
There may be some delay in data statistics. You can try again after a few minutes to see if the warning still appears.

**Comment by gbrew** [comment]
@vvvvvv1vvvvvv I did wait at least 20 minutes, with multiple transfer attempts and at least two restarts of rabby. So I don't think it was due to a delay.

I eventually just did the USDC transfer despite the warning, and I was then able to transfer DAI with no warning. So maybe the issue is specific to USDT?

Also, I forgot to mention this above, but I was seeing this with both regular rabby and rabby desktop.
